server/server.js

```diff
@@ -1,15 +1,150 @@
 const express = require("express");
+const fs = require("fs");
 const app = express();
-const port = process.env.PORT || 5000;
+const cors = require("cors");
+const port = process.env.PORT || 5001;
+const { Pool } = require("pg");
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
+require("dotenv").config();
+
+app.use(express.json());
+app.use(cors());
+
+// Create a PostgreSQL pool
+const db = new Pool({
+  user: process.env.DB_USERNAME,
+  host: process.env.DB_HOST,
+  database: process.env.DB_DATABASE,
+  port: process.env.DB_PORT,
+  password: process.env.DB_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+// const db = new Pool({
+//   connectionString: process.env.DB_CONNECTION_STRING,
+// });
 
-// Store and retrieve your videos from here
-// If you want, you can copy "exampleresponse.json" into here to have some data to work with
 let videos = [];
 
-// GET "/"
-app.get("/", (req, res) => {
-  // Delete this line after you've confirmed your server is running
-  res.send({ express: "Your Backend Service is Running" });
+// Store and retrieve videos
+fs.readFile("./exampleresponse.json", "utf8", (err, data) => {
```

Comment:
Is the JSON file to provide some default videos? That works fine for a demo app. 
It'd be more common to 'seed' the database with default data in a 'real' production app.